pkg/apis/options/options.go

```diff
@@ -36,7 +36,7 @@ type Options struct {
 	TLSKeyFile         string   `flag:"tls-key-file" cfg:"tls_key_file"`
 
 	AuthenticatedEmailsFile  string   `flag:"authenticated-emails-file" cfg:"authenticated_emails_file"`
-	KeycloakGroup            string   `flag:"keycloak-group" cfg:"keycloak_group"`
+	KeycloakGroups           []string `flag:"keycloak-groups" cfg:"keycloak_groups"`
```

Comment:
This needs to stay singular from the flag POV (singular with multiple flags, `cfg` is plural for naming in `cfg` & environment variables)